contentcuration/contentcuration/tasks.py

```diff
@@ -292,24 +290,53 @@ def sendcustomemails_task(subject, message, query):
 
 
 type_mapping = {
-    "duplicate-nodes": {"task": duplicate_nodes_task, "progress_tracking": True},
-    "move-nodes": {"task": move_nodes_task, "progress_tracking": False},
-    "delete-node": {"task": delete_node_task, "progress_tracking": False},
-    "export-channel": {"task": export_channel_task, "progress_tracking": True},
-    "sync-channel": {"task": sync_channel_task, "progress_tracking": True},
-    "get-node-diff": {"task": generatenodediff_task, "progress_tracking": False},
+    "duplicate-nodes": duplicate_nodes_task,
+    "move-nodes": move_nodes_task,
+    "delete-node": delete_node_task,
+    "export-channel": export_channel_task,
+    "sync-channel": sync_channel_task,
+    "get-node-diff": generatenodediff_task,
+    "calculate-resource-size": calculate_resource_size_task,
```

Comment:
We could get rid of this in the future, if we align the `task_name` sent to `create_async_task()` with the names of the tasks we define for Celery. Right now, they're slightly different

Reply:
Yeah, I always found this mapping odd, when Celery explicitly has you define a name for the task.